src/setupTests.js

```diff
@@ -0,0 +1 @@
+// add logic here that needs to be run before any tests are run.
```

Comment:
This file should move to a top-level `test` folder and the reference in `scripts/test.js` should me modified.